js/src/helpers/replacementVariableHelpers.js

```diff
@@ -83,6 +83,27 @@ export function createLabelFromName( name ) {
 	return name[ 0 ].toUpperCase() + name.slice( 1 );
 }
 
+/**
+ * Pushes a new replacement variable from an action into the replacementVariables array.
+ * Creates a label from the replacement variable name when no label is supplied.
+ *
+ * @param {array}  replacementVariables The current replacement variable list
+ * @param {Object} action               The UPDATE_REPLACEMENT_VARIABLE action.
+ * @param {string} action.name          The name of the replacement variable.
+ * @param {string} [action.label]       The label of the replacement variable (optional).
+ * @param {*} action.value              The value of the replacement variable.
```

Comment:
perhaps line up the last param, when you already did it so neatly with the previous ones.